agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AfterAgentListener.java

```diff
@@ -4,20 +4,46 @@
 package com.microsoft.applicationinsights.agent.internal.init;
 
 import com.google.auto.service.AutoService;
+import com.microsoft.applicationinsights.agent.internal.configuration.Configuration;
 import com.microsoft.applicationinsights.agent.internal.httpclient.LazyHttpClient;
+import com.microsoft.applicationinsights.agent.internal.profiler.ProfilingInitializer;
+import com.microsoft.applicationinsights.agent.internal.telemetry.TelemetryClient;
 import io.opentelemetry.javaagent.extension.AgentListener;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @AutoService(AgentListener.class)
 public class AfterAgentListener implements AgentListener {
 
+  private static final Logger logger = LoggerFactory.getLogger(AfterAgentListener.class);
+
   @Override
   public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredOpenTelemetrySdk) {
     // only safe now to make HTTPS calls because Java SSL classes
     // trigger loading of java.util.logging (starting with Java 8u231)
     // and JBoss/Wildfly need to install their own JUL manager before JUL is initialized.
     LazyHttpClient.safeToInitLatch.countDown();
 
-    PerformanceCounterInitializer.initialize(FirstEntryPoint.getConfiguration());
+    Configuration configuration = FirstEntryPoint.getConfiguration();
+    TelemetryClient telemetryClient = TelemetryClient.getActive();
```

Comment:
TelemetryClient is only used in the if at line 33. The code would be a little more readable for me if the TelemetryClient local variable was declared just before the if.